janitor/index.d.ts

```diff
@@ -46,20 +46,70 @@ export class Janitor<U extends object | void = void> {
 	 */
 	public Remove(index: keyof U): this;
 
+	/**
+	 * Removes an object from the Janitor without running a cleanup.
+	 *
+	 * ```ts
+	 * import { Janitor } from "@rbxts/janitor";
+	 *
+	 * const Obliterator = new Janitor<{ Function: () => void }>();
+	 * Obliterator.Add(() => print("Removed!"), true, "Function");
+	 *
+	 * Obliterator.RemoveNoClean("Function"); // Does not print.
+	 * ```
+	 * @param index The index you want to remove.
```

Comment:
To be consistent with docs let's change this to
```suggestion
	 * @param index The index you are removing.
```